cosmo_tester/test_suites/cli/linux_test.py

```diff
@@ -94,7 +77,8 @@ def test_cli_on_linux(linux_cli_tester, module_tmpdir, ssh_key, logger):
     cli_host.run_command('chmod 500 {}'.format(remote_uninstall_script_path))
 
     example = get_example_deployment(
-        manager_host, ssh_key, logger, linux_cli_tester['url_key'])
+        manager_host, ssh_key, logger, linux_cli_tester['url_key'],
+        test_config)
```

Comment:
It can't be used as a fixture because it needs things like IP addresses from VMs, and some of the tests invoke the VM creation themselves, which would mean that the IP address wouldn't be available.
I was considering reworking the VM creation logic to remove the need for this, but that's a bigger task than fits in this.